### PR TITLE
Replacing amd64 with $CONFIGURED_ARCH

### PR DESCRIPTION
**Pull request by Sabareesh-Kumar-Anandan**
Signed-off-by: Sabareesh Kumar Anandan <sanandan@marvell.com>

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/make -f  
 %:  
+	sed -i "s/Architecture: .*/Architecture: $(CONFIGURED_ARCH)/g" debian/control
```
Comment (antony-rheneus) [inline debian/rules:3]: Not like this, Change in debian/control, Archiecture : any, that would be suffice.

File: Makefile [hunk 2]
```diff
@@ -36,7 +36,7 @@ endif
 
 INSTALL := /usr/bin/install
 
-MAIN_TARGET = sonic-mgmt-framework_1.0-01_amd64.deb
+MAIN_TARGET = sonic-mgmt-framework_1.0-01_$(CONFIGURED_ARCH).deb
```
Comment (antony-rheneus) [inline Makefile:39]: Initialize as below to initialize CONFIGURED_ARCH if incase it was not declared
CONFIGURED_ARCH ?= amd64

File: debian/rules (hunk 1 above)
Comment (Sabareesh-Kumar-Anandan) [reply debian/rules]: Done

File: Makefile (hunk 2 above)
Comment (Sabareesh-Kumar-Anandan) [reply Makefile:41]: done

**Pull request merged**